client/tasks.py

```diff
@@ -0,0 +1,49 @@
+from logging import getLogger
+
+from celery.task import task
+from django.conf import settings
+from openedx.features.openedx_nodebb_discussion.client.categories import NodeBBCategory
+from openedx.features.openedx_nodebb_discussion.client.users import NodeBBUser
+
+RETRY_DELAY = 10
```

Comment:
I believe `RETRY_DELAY` should be configurable from the settings file. Open edX already has a celery retry delay config. See if you can use that.